xsd/xsd.go

```diff
@@ -110,6 +111,19 @@ type Schema struct {
 	Doc string
 }
 
+// MergeTypes adds the types from the incoming schema into
+// the receiving schema, returns error if the schemas have
+// different target namespaces.
+func (s *Schema) MergeTypes(incoming *Schema) error {
+	if !strings.EqualFold(s.TargetNS, incoming.TargetNS) {
```

Comment:
I was under the impression that XML namespaces are case sensitive. What is the reason for using EqualFold over == here?